packages/cxl-ui/src/components/cxl-app-layout.js

```diff
@@ -59,7 +59,7 @@ export class CXLAppLayoutElement extends LitElement {
   wide;
 
   // Device Detector media query.
-  _wideMediaQuery = '(min-width: 750px)';
+  _wideMediaQuery = '(min-width: 768px)';
```

Comment:
Could you add more context @kertuilves? Why it was 750 and now is 768?

Reply:
I changed it because this layout query and css media queries weren't matching and it made some things to behave weirdly on screen sizes 750-768px. But I changed it back and found another solution.

Reply:
OK, cool 👍  It looked risky as it seem to affect all layout

Reply:
This value needs better documentation, must not be altered for trivial reasons. It's linked to vaadin-context-menu. Vaadin has an open issue for documenting all magic numbers like this, but it's not super high on their priority list.